HabitLogger.BBualdo/DatabaseLibrary/DbContext.cs

```diff
@@ -0,0 +1,636 @@
+﻿using DatabaseLibrary.Models;
+using Microsoft.Data.Sqlite;
+using System.Globalization;
+
+namespace DatabaseLibrary;
+
+public class DbContext
+{
+  private SqliteConnection _Connection { get; set; }
```

Comment:
Using one connection is fine for this application, but for larger applications with multiple users it's not best practice. Here's ChatGPTs explanation:

> The reason is that database connections are not thread-safe. If multiple threads try to use the same connection at the same time, it can lead to unpredictable results and potential data corruption.

> A better approach is to use a connection pool. A connection pool manages a set of database connections. When a connection is needed, it is obtained from the pool. When it's no longer needed, it's returned to the pool to be reused later. This approach is more efficient and safer for concurrent use.

> In .NET, connection pooling is typically handled automatically when you create a new connection. The pool is managed by the .NET runtime, and connections are reused when possible. So, instead of storing a single SqliteConnection instance in your class, you would create a new connection each time you need to interact with the database, and dispose of it when you're done.